docs/swagger.yaml

```
@@ -2240,7 +2241,8 @@ paths:
                     "start": "2015-07-17T12:00:00Z",
                     "start_display": "July 17, 2015",
                     "start_type": "timestamp",
-                    "pacing": "instructor"
+                    "pacing": "instructor",
+                    "certificate_available_date": "2015-08-14T00:00:00Z"
```

Can we get away with this?

```suggestion
                    "certificate_available_date": "2015-08-14T00:00:00Z", 
```

---

`,` at the last row, I think is invalid in JSON format and also it was not present previously that's why I've not added this at the last line